src/Tribe/Commerce/PayPal/Gateway.php

```diff
@@ -659,7 +669,7 @@ public function get_paypal_cart_api_url( $post_id ) {
 		 * @param string $cart_url
 		 * @param array  $post_data
 		 */
-		$url = apply_filters( 'tribe_tickets_commerce_paypal_gateway_add_to_cart_redirect', $url, $cart_url, [] );
+		$url = apply_filters( 'tribe_tickets_commerce_paypal_gateway_add_to_cart_redirect', $cart_url, $cart_url, [] );
```

Comment:
Was this supposed to change? If so why bother setting `$url` above?

Reply:
Backwards compatibility to override the URL, but no longer using the home_url() URL and instead using the full PayPal cart URL